intel-retail/automated-self-checkout
# fix: Enabling render mode on gst pipelines

## PR Checklist



## What are you changing?


## Issue this PR will close

close: #**issue_number**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

1. Re-download models in case you haven't.  **make clean-models**
2. make run-camera-simulator
3. PIPELINE_PROFILE="gst" RENDER_MODE=1 sudo -E ./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0
4. Try with RENDER_MODE=0 to see no ouput.

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
